packages/@tinacms/toolkit/src/packages/form-builder/FormBuilder.tsx

```diff
@@ -254,8 +285,18 @@ export const FullscreenFormBuilder: FC<FormBuilderProps> = ({
         onSubmit={tinaForm.onSubmit}
       >
         {({ handleSubmit, pristine, invalid, submitting }) => {
+          const cannotSubmit = pristine || submitting || invalid
```

Comment:
This conditional is a bit different than the above cannotSubmit. Is that intentional?

Reply:
Yes, follows the disabled state of each save button from the two forms. FullscreenFormBuilder seems to have a slightly simpler version:
<img width="682" alt="Screenshot 2023-01-03 at 9 19 36 AM" src="https://user-images.githubusercontent.com/5762581/210408341-df9b5473-720f-41af-acb0-763ac88b4291.png">
<img width="829" alt="Screenshot 2023-01-03 at 9 19 21 AM" src="https://user-images.githubusercontent.com/5762581/210408345-0e24f0ee-6a5c-4c6a-a4e0-fc2318c632fd.png">